python_hooks/disallowed_identifiers.py

```diff
@@ -0,0 +1,84 @@
+import argparse
+import ast
+import enum
+import sys
+
+from python_hooks.utills.ignore_check import IdentifierCheck
+from python_hooks.utills.ignore_check import ignore_check
+
+DISALLOWED_MESSAGE = "Flagged {identifier} {name} in {filename}:{line} column {col_offset}. Replace with {replacement}"
```

Comment:
changed the format here, the `:line` is nice for warp users who can click the output message to open IDE file @ line.